cardinal/server/server.go

```diff
@@ -46,6 +53,152 @@ const (
 	getSignerForPersonaStatusAssigned  = "assigned"
 )
 
+// NewSwaggerHandler instantiates handler function for creating a swagger server that validates itself based on a swagger spec.
+func NewSwaggerHandler(w *ecs.World, pathToSwaggerSpec string, opts ...Option) (*Handler, error) {
```

Comment:
Did you consider making the pathToSwaggerSpec parameter into an Option?

Reply:
Yeah I don't know, Seems more straightforward as a string in my opinion. I'm not to tied to either way.